96jonesa/AutoBought
# Deleted react-scripts folder; Added patch-package dependency to secure local host

How to test:
Delete your current node_modules folder.
Do 'yarn install'.
Check to see that line 65 in node_modules/react-scripts/scripts/start.js has '127.0.01' instead of '0.0.0.0' or try to access the frontend from another device.
